docs/docker.md

```diff
@@ -21,7 +21,8 @@ Feel free to comment out the apps you do not need.
 Create a configuration directory on your host machine:
 ```
 git clone -b docker-master https://github.com/georchestra/datadir.git
-sudo mv datadir /etc/georchestra
+sudo mkdir /etc/georchestra
+sudo mv datadir/* /etc/georchestra/.
```

Comment:
I dislike this command since it fails to copy the `.git` folder, which is useful for those who will want to version their datadir.